Prototope/Layer.swift

```diff
@@ -253,13 +258,23 @@ public class Layer: Equatable, Hashable {
 	}
 
 	/** The scaling factor of the layer along the x dimension. Defaults to 1. */
-	public var scaleX: Double = 1 {
-		didSet { updateTransform() }
+	public var scaleX: Double {
+        get {
+            return layer.valueForKeyPath("transform.scale.x") as Double
```

Comment:
@saniul: You've totally got it right. I haven't actually written this out, but I intend to deal with this dichotomy through the distinction between `layer.animators.x.target` and `layer.x`. i.e. the first is "like the model value" and the second is "like the presentation value."

Yeah, we still have this semi-vestigial `Layer.animateWithDuration` method which doesn't use the POP animators, but I really don't like that API anyway. Gotta merge it with the dynamic animators as a convenience or otherwise make it go through the same paths. Haven't thought it through.